src/components/Sidebar/Participants/ParticipantsTab.vue

```
@@ -166,34 +166,50 @@ export default {
 		}, 250),
 
 		async fetchSearchResults() {
-			const response = await searchPossibleConversations(this.searchText)
-			this.searchResults = response.data.ocs.data
-			this.getParticipants()
-			this.contactsLoading = false
+			try {
+				const response = await searchPossibleConversations(this.searchText)
+				this.searchResults = response.data.ocs.data
+				this.getParticipants()
+				this.contactsLoading = false
+			} catch (exeption) {
+				console.debug(exeption)
```

This is an error and is failing hard.
This should be at least a `console.error` :)

Do you do anything to warn the user?

---

Ok for the error. No we don't warn for now

---

Well you should, otherwise the user don't know what the hell is happening :p

---

So what should we do here? cc @nickvergessen

---

`OCP.Toast.error(t('spreed', 'An error occurred while performing the search'))` ?

---

> ?

I'm asking you :sweat_smile:

---

Done